app/components/provider_interface/location_presentation_component.rb

```diff
@@ -0,0 +1,22 @@
+module ProviderInterface
+  class LocationPresentationComponent < ActionView::Component::Base
+    attr_reader :application_choice
+
+    def initialize(application_choice:)
+      @application_choice = application_choice
+    end
+
+    def course_option
+      application_choice.offered_course_option || application_choice.course_option
+    end
+
+    def site
+      course_option.site
+    end
+
+    def site_address_or_name
+      address = site.full_address
+      address.presence || site.name
```

Comment:
Is it odd not to show the site name, only the address, in this case?

Reply:
Doing name + address, or just name, if address is missing